src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1587,6 +1617,8 @@ def load_arguments(self, _):
     with self.argument_context('restore-point wait') as c:
         c.argument('restore_point_name', options_list=['--name', '-n', '--restore-point-name'],
                    help='The name of the restore point.')
+        c.argument('expand', help="The expand expression to apply on the operation. 'InstanceView' retrieves "
+                                  "information about the run-time state of a restore point. 'instanceView' Default value is None.")
```

Comment:
Could we only add these help message in `_help.py` without registering these parameters?

Reply:
These params exist in help messages of current stable version. If we unregister these params manually, it would bring a breaking change to our CLI.